### PR TITLE
Introduce stylelint

### PR DESCRIPTION
**Pull request by lukemelia**

### CODE REVIEW

File: tests/dummy/app/css/components/isolated-collection.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* stylelint-disable */
```
Comment (Aierie) [inline tests/dummy/app/css/components/isolated-collection.css:1]: Here

File: tests/dummy/app/css/components/version-history.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* stylelint-disable */
```
Comment (Aierie) [inline tests/dummy/app/css/components/version-history.css:1]: Here

**Review by Aierie — APPROVED** [review]
Thanks for working on this! This looks good, doesn't look like stylelint aggressively introduces new changes/practices. Having blocks of declarations move around was a little jarring in some places but I think it's a good thing so we can read css from top to bottom with the expectation that the bottom will take priority (https://stylelint.io/user-guide/rules/no-descending-specificity).

Why did we have stylelint disabled for these 2 files?

**Comment by lukemelia** [comment]
>Why did we have stylelint disabled for these 2 files?

Ah, yes. Those files had so many dozens of lint complaints that I didn't have time to start untangling them!

**Comment by Aierie** [comment]
> > Why did we have stylelint disabled for these 2 files?
> 
> Ah, yes. Those files had so many dozens of lint complaints that I didn't have time to start untangling them!

I see! Seems okay to leave them as is considering they're not Boxel components.

**Pull request merged**